src/main/java/com/google/devtools/build/lib/runtime/UiOptions.java

```diff
@@ -178,6 +234,17 @@ public ProgressModeConverter() {
       help = "Report all events known to the Bazel UI.")
   public boolean experimentalUiDebugAllEvents;
 
+  @Option(
+      name = "ui_event_filters",
+      converter = EventFiltersConverter.class,
+      defaultValue = "null",
+      metadataTags = {OptionMetadataTag.HIDDEN},
+      documentationCategory = OptionDocumentationCategory.UNDOCUMENTED,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help = "Specifies which events to show in the UI.",
```

Comment:
Can we expand the help text a bit here and mention the + /  - syntax and some examples of what kind of events can be specified here? (I had to look up the EventKind class to figure out what the supported events are.)